coredb-operator/tests/integration_tests.rs

```diff
@@ -100,7 +100,7 @@ mod test {
         let replicas = 1;
 
         // Timeout settings while waiting for an event
-        let timeout_seconds_start_pod = 60;
+        let timeout_seconds_start_pod = 90;
```

Comment:
pod startup is slowing down. I don't think its completely related to this PR and it might be the reason for our flaky tests. I'm looking into it now.

Reply:
There may be some issue with no concurrency, like the operator only handling one coredb at a time. I think I recall noticing that last time I incremented this value.